HoneyBadgers.Logic/Services/FavoriteMoviesService.cs

```diff
@@ -50,15 +51,56 @@ public List<MovieViewModel> GetAllMoviesAsMovieViewModels()
 
             return movieViewModels;
         }
-        public void AddFavorite(string id)
+
+        public List<FavoriteMoviesViewModel> GetAllFavoriteMoviesViewModels()
+        {
+            var movieViewModels = new List<FavoriteMoviesViewModel>();
+            var movies = _movieService.GetAll();
+            var favorite = GetAllFavoriteMovieId();
+            foreach (var movie in movies)
+            {
+                foreach (var f in favorite)
+                {
+                    
```

Comment:
Po co ten foreach?

Reply:
też wydaje mi się że te dwa foreach są trochę przekombinowane

Reply:
Ten akurat jest pusty :) body jest pusty :o
```
foreach (var f in favorite)
                {

                }
```

Reply:
Ten pusty to pozostałość po testowaniu, sprawdzałem sobie coś i zostało. usunięte :)